Could not initialize collection 'la_plata_assessor': Collection [la_plata_assessor] does not exists
INFO:apis.search.search_engine:Connecting to ChromaDB...
INFO:apis.search.search_engine:Connected to collection 'la_plata_county_code': 1298 documents
WARNING:apis.search.search_engine:Could not initialize collection 'la_plata_assessor': Collection [la_plata_assessor] does not exists
INFO:apis.search.search_engine:Successfully initialized 1 collections
INFO:__main__:Search system initialized successfully
INFO:__main__:Starting Flask server in development mode...
 * Serving Flask app 'apis.search.app_factory'
 * Debug mode: on
Address already in use
Port 8000 is in use by another program. Either identify and stop that program, or start the server with a different port.
